opentasks-contract/src/main/java/org/dmfs/tasks/contract/TaskContract.java

```diff
@@ -1035,6 +1035,14 @@ public static Uri getSearchUri(String authority, String query)
          */
         String INSTANCE_DURATION = "instance_duration";
 
+        /**
+         * The start of the original instance as specified in the master task. For non-recurring task instances this equals the value of {@link
+         * #INSTANCE_START}, except that `null` values are represented as `0`.
+         * <p>
+         * For recurring tasks, these are the timestamps which have been derived from the recurrence rule or dates, except those specified as exdates.
```

Comment:
`exdates` is not a typo. It's iCalendar terminology for `Exclusion dates`, i.e. dates at which a recurring event does not occur even if the rule says so.

Using `0` instead of `null` makes it much easier to handle, because we can just compare the values without doing any `null` checks. Semantically it doesn't really matter, because recurring events can't have `null` here, by definition.

Reply:
ok, thanks